docs/pages/database-access/getting-started.mdx

```diff
@@ -0,0 +1,157 @@
+   ]
+}
+```
+
+This policy allows any database account to connect to the Aurora instance specified
+with resource ID using IAM auth.
+
+<Admonition type="note" title="Resource ID">
+Database resource ID is shown on the Configuration tab of a particular
+database instance in RDS control panel, under "Resource id". For regular
+RDS database it starts with `db-` prefix. For Aurora, use the database
+cluster resource ID (`cluster-`), not individual instance ID.
+</Admonition>
+
+Finally, connect to the database and create a database account with IAM auth
+support (or update an existing one). Once connected, execute the following
+SQL statements to create a new database account and allow IAM auth for it:
+
+```sql
+CREATE USER alice;
+GRANT rds_iam TO alice;
+```
+
+For more information about connecting to the PostgreSQL instance directly,
+see Amazon [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToPostgreSQLInstance.html).
+
+## Step 2/3. Setup Teleport
+
+Teleport Database Access is available starting from `6.0.0` release.
+
+Download the appropriate version of Teleport for your platform from the table
+below, or visit our [downloads page](https://goteleport.com/teleport/download).
+
+{!docs/pages/preview/releases-table.mdx!}
+
+<Admonition type="warning" title="Version Warning">
+Note, pre-releases are not suitable for production usage!
+</Admonition>
+
+Start Teleport using the following command and point it to your Aurora database
+instance. Make sure to update the database endpoint and region appropriately.
+
+```shell
+sudo teleport start \
+  --roles=proxy,auth,db \
+  --db-name=aurora \
+  --db-protocol=postgres \
+  --db-uri=postgres-aurora-instance-1.abcdefghijklm.us-west-1.rds.amazonaws.com:5432 \
+  --db-aws-region=us-west-1
+```
+
+<Admonition type="note" title="AWS Credentials">
+The node where the Teleport process is started should have AWS credentials
+configured with the policy from [step 1](#step-13-setup-aurora).
+</Admonition>
+
+Create a Teleport user that is allowed to connect to a particular database
+(e.g. `postgres`) within the Aurora instance as a particular database account
+(e.g. `alice`).
+
+```shell
+sudo tctl users add alice root \
+  --db-names=postgres \
+  --db-users=alice
+```
+
+## Step 3/3. Connect
+
+Now that Aurora is configured with IAM authentication, Teleport is running and
+the local user is created, we're ready to connect to the database.
+
+Log into Teleport with the user we've just created. Make sure to use `tsh`
+version `6.0.0` or newer that includes Database Access support.
+
+For simplicity, we're using an `--insecure` flag to accept Teleport's
+self-signed certificate. For production usage, make sure to configure proxy
+with a proper certificate/key pair. See Teleport's general
+[quickstart guide](../quickstart.mdx#step-1c-configure-domain-name-and-obtain-tls-certificates-using-lets-encrypt).
+
+```shell
+tsh login --insecure --proxy=localhost:3080 --user=alice
```

Comment:
Let's never promote the use of `insecure` flag.

Reply:
@klizhentas I agree we should not promote it, which is why I included that warning above. I was pretty conflicted myself adding it here but this was the easiest way to try out db access locally I could come up with - just a single command basically. I can think of two other options:

1. Just remove flag from here and add a note telling folks to install a proper certificate, and maybe mention under a warning that if they don't have ability to provide a valid cert they can use insecure flag but it's unadvised in anything other than test setting.
2. Adjust this to use ACME, but in this case they would have to setup DNS records for the cluster name and deploy Teleport on the node where this DNS points, as far as I understand, in order to pass ACME challenge?

Both of these options are more secure of course but result in a bit worse getting started experience i.e. you can't just easily launch it on your laptop in 5 minutes unless you already have certs/DNS.

I'm okay making the changes - just let me know which option you would prefer? Probably ACME since we added it specifically for better quickstarts experience?

And btw FWIW we also use insecure flag [elsewhere](https://goteleport.com/teleport/docs/enterprise/quickstart-enterprise/#logging-in) with a warning. Maybe we should consider updating it there as well then.

Reply:
@klizhentas Anyway, I've refactored this to use ACME and removed the use of insecure flag completely.